templates/location.html

```diff
@@ -17,15 +17,10 @@
 <h1>Location - {{ locdet["name"] }}</h1>
 <script>
 	$(document).ready(function() {
-		$("#success-alert").hide();
         getData("{{ loc_id }}", "{{ rack_num }}");
         setInterval(function() {
-			$("#success-alert").hide();
-            getData("{{ loc_id }}", "{{ rack_num }}")
-			$("#success-alert").fadeTo(2000, 500).slideUp(500, function() {
-      			$("#success-alert").slideUp(500);
-    		});
-        },60000);
+            getData("{{ loc_id }}", "{{ rack_num }}");
+        },600000);
     });
 </script>
```

Comment:
Put this script into a js file in your static folder, add an extrajs block to your template and link it in the template.

Example:
_base.html_
```
# At bottom of body element
{% block extrajs %}
{% endblock %}
```
_location.html_
```
# After content block
{% block extrajs %}
<script src="{{ url_for('static', 'js/function.js') }}">
{% endblock %}
```